FABulous/FABulous.py

```diff
@@ -869,10 +870,50 @@ def do_gen_geometry(self, *vargs):
         if 4 <= padding <= 32:
             self.fabricGen.genGeometry(padding)
             logger.info("Geometry generation complete")
-            logger.info(f"{geomFile} can now be imported into the FABulous Editor")
+            logger.info(f"{geomFile} can now be imported into FABulator")
         else:
             logger.error("padding has to be between 4 and 32 inclusively!")
 
+    def do_start_FABulator(self, *ignored):
+        """Starts FABulator if an installation can be found.
+        If no installation can be found, a warning is produced.
+
+        Usage:
+            start_FABulator
+
+        Parameters
+        ----------
+        *ignored : tuple
+            Ignores additional arguments.
+
+        """
+        logger.info("Checking for FABulator installation")
+        fabulatorRoot = os.getenv("FABULATOR_ROOT")
+        if fabulatorRoot is None:
+            logger.warning("FABULATOR_ROOT environment variable not set.")
+            logger.warning(
+                "Install FABulator and set the FABULATOR_ROOT environment variable to use this feature."
```

Comment:
I suggest adding a link to the FABulator directory so people can find the link to it.

Reply:
Sorry, I'm not sure if I understand

Reply:
Something like "Install FABulator (https://github.com/FPGA-Research-Manchester/FABulator) and ..."